src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
+import core.basesyntax.vehicles.Bulldozer;
+import core.basesyntax.vehicles.Excavator;
+import core.basesyntax.vehicles.Machine;
+import core.basesyntax.vehicles.Truck;
+
 public class MainApp {
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[]{new Truck(), new Bulldozer(), new Excavator()};
 
+        for (Machine machine : machines) {
+            machine.doWork();
+            machine.stopWork();
+        }
+    }
```

Comment:
Remember to use abstract references instead of specific ones where possible. In this case, you've done it correctly by declaring the array with the abstract type 'Machine'. This allows for polymorphism and is a good practice.